Source/WebKit/Shared/Extensions/WebExtensionContextParameters.h

```diff
@@ -47,6 +47,7 @@ struct WebExtensionContextParameters {
 
     double manifestVersion;
     bool testingMode;
+    bool isSessionStorageAllowedInContentScripts;
```

Comment:
These all should have default initializers, so they don't get random memory if they are not explicitly set.

```
double manifestVersion { 0 };
bool testingMode { false };
bool isSessionStorageAllowedInContentScripts { false };
```